src/runway/path.py

```diff
@@ -0,0 +1,119 @@
+""" Runway configuration 'path' settings """
+import os
+import logging
+import six
+
+from .sources.git import Git
+
+LOGGER = logging.getLogger('runway')
+
+
+class Path(object):
+    """ Runway configuration 'path' settings object """
+
+    def __init__(self, module, env_root, cache_dir=None):
+        """ Initialize. """
+        if not cache_dir:
+            cache_dir = os.path.expanduser("~/.runway_cache")
+
+        self.env_root = env_root
+        self.cache_dir = cache_dir
+        self.source, self.uri, self.location, self.options = self.parse(module)
+        self.module_root = self.__get_module_root_dir(module)
+
+    def parse(self, module):
+        """ Retrieve the source and location of the path variable. """
+        source = 'local'
+        uri = ''
+        location = ''
+        options = ''
+
+        split_source_location = module.get('path').split('::')
+
+        # Local path
+        if len(split_source_location) != 2:
+            location = split_source_location[0]
+            return [source, uri, location, options]
+
+        source = split_source_location[0]
+        temp_location = split_source_location[1]
+
+        uri, location = self.__parse_uri_and_location(temp_location)
+        location, options = self.__parse_location_and_options(location)
+
+        return [source, uri, location, options]
```

Comment:
When returning multiple values, it should be a tuple (immutable) instead of a list (mutable).

This can be done with this

```suggestion
        return source, uri, location, options
```

which is like returning this but you don't need the parenthesis

```python
        return (source, uri, location, options)
```